tests/qpu/test_leaphybrid_common.py

```diff
@@ -123,8 +124,8 @@ def test_sampleset_wait_id_availability(self):
             pid_post = ss.wait_id()
             self.assertEqual(pid, pid_post)
 
-    @unittest.skipIf(tuple(map(int, cc_version.split('.'))) < (0, 13, 5),
-                     "'dwave-cloud-client>=0.13.5' required")
+    @unittest.skipUnless(cc_version in SpecifierSet('>=0.13.5', prereleases=True),
+                         "'dwave-cloud-client>=0.13.5' required")
```

Comment:
This should be our go-to way of testing package versions. Manual parsing is too fragile in today's world. :smile: